test/node/module/authenticate/setClientCertByPath.test.js

```diff
@@ -162,7 +162,7 @@ describe('Connection module', () => {
       const auth = new Auth()
         .setPasswordAuth(paramPasswordAuth)
         .setClientCertByPath(paramClientCertByPath);
-      const conn = new Connection(common.DOMAIN, auth);
+      const conn = new Connection({domain: common.DOMAIN, auth: auth});
       const appModule = new App(conn);
```

Comment:
App module will be changed to single object params. Update App module first, then update this PR